assets/pages/bundles-landing/components/Bundles.jsx

```diff
@@ -10,10 +10,14 @@ import type { ListItem } from 'components/featureList/featureList';
 import CtaLink from 'components/ctaLink/ctaLink';
 import Bundle from 'components/bundle/bundle';
 import ContribAmounts from 'components/contribAmounts/contribAmounts';
+import PayPalContributionButton from 'components/payPalContributionButton/payPalContributionButton';
 import type { Contrib, Amounts, ContribError } from 'helpers/contributions';
 import type { IsoCountry } from 'helpers/internationalisation/country';
 import type { IsoCurrency, Currency } from 'helpers/internationalisation/currency';
 import type { Campaign } from 'helpers/tracking/acquisitions';
+import type { Participations } from 'helpers/abtest';
```

Comment:
Nice! Much cleaner than: `'../../../helpers/abtest`... 😯

Reply:
I think intellij adds that!